website/content/docs/jekyll.md

```diff
@@ -0,0 +1,463 @@
+---
+title: Jekyll
+weight: 20
+group: guides
+---
+
+## Introduction
+
+This section will help you integrate Netlify CMS with a new or existing Jekyll project.
+
+[Jekyll](https://jekyllrb.com/) is a blog-aware static site generator built with Ruby. [Github Pages](https://pages.github.com/) are powered by Jekyll, making it a popular choice for developer blogs and project pages.
+
+If you're starting a new project, the fastest route to publishing on a Jekyll website with Netlify CMS is to [deploy a template on Netlify](https://templates.netlify.com/).
+
+This guide aims to be helpful for users in the following situations.
+
+- You have an existing Jekyll project and you want to add Netlify CMS
+- You have a Jekyll project with Netlify CMS integrated and want to make updates beyond adding content
+- You are starting a new Jekyll + Netlify CMS project and you want to start from scratch so you know how everything fits togeter
+
+## Setup
+
+### Create a Jekyll Site
+
+This guide will use the blog you get if you follow the [really excellent official Jekyll step by step tutorial](https://jekyllrb.com/docs/step-by-step/01-setup/) as a starting point. If you're new to Jekyll - I recommended you start by following the tutorial so you know your way around your new blog. Otherwise [you can clone this repo](https://github.com/adamwatters/jekyll-tutorial-with-netlify-cms/tree/without-cms) and checkout the `without-cms` branch.
+
+![Jekyll tutorial blog screenshot](https://www.netlifycms.org/img/screenshot-jekyll-tutorial-blog?raw=true)
+
+### Hosting and Serving
+
+You have lots of options for hosting and serving your project, but to for the sake of simplicity this guide will assume you [follow this guide](https://www.netlify.com/blog/2015/10/28/a-step-by-step-guide-jekyll-3.0-on-netlify/) to create a remote git repository on Github and give Netlify access to build and serve the project when you push changes.
+
+### Why Not Stop Here?
```

Comment:
This section is a little out of scope, can probably be dropped.

Reply:
Done 👍